### DISCUSSION TITLE
Any way to disable the SDK trace logging?

### DISCUSSION THREAD

**Discussion by Luminoth** [awslabs/aws-sdk-rust#652] [General]
If I'm using tracing in my application, that ends up also enabling tracing in the AWS SDK and I would like to not have those logs. For example this stuff:

```
2022-10-28T22:14:27.947430Z  INFO provide_credentials{...}
2022-10-28T22:14:27.947664Z  INFO provide_credentials{...}
2022-10-28T22:14:27.947873Z  INFO provide_credentials{...}
```

I can't find anything that will let me stop logging that, or a way to move it to the Debug or Trace level where it would probably make more sense for us.

**Comment by jdisanti** [comment]
This seems like a bug to me, so I opened https://github.com/awslabs/aws-sdk-rust/issues/653 to track it.

With `tracing-subscriber`, you should be able to configure the fmt subscriber to use a `warn` level for the SDK as a workaround for now.
